app/src/main/java/com/simple/checkstyledemo/MainActivity.java

```diff
@@ -1,13 +1,20 @@
 package com.simple.checkstyledemo;
 
-import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
 
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        int i = 3;
+        test(i);
+    }
+
+    private void test(int a) {
+        if (a == 3) return;
+        System.out.println(a);System.out.println(a);System.out.println(a);System.out.println(a);System.out.println(a);
```

Comment:
**Reporter**: Checkstyle
**Rule**: com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck
**Severity**: ERROR
**File**: app/src/main/java/com/simple/checkstyledemo/MainActivity.java L18

&apos;;&apos; is not followed by whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*